watertap/unit_models/coag_floc_zo_jartest.py

```diff
@@ -0,0 +1,623 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+# Import Pyomo libraries
+from pyomo.environ import (Block,
+                           Set,
+                           Var,
+                           Param,
+                           Expression,
+                           Suffix,
+                           NonNegativeReals,
+                           Reference,
+                           value,
+                           units as pyunits)
+from pyomo.common.config import ConfigBlock, ConfigValue, In
+
+# Import IDAES cores
+from idaes.core import (ControlVolume0DBlock,
+                        declare_process_block_class,
+                        MaterialBalanceType,
+                        EnergyBalanceType,
+                        MomentumBalanceType,
+                        UnitModelBlockData,
+                        useDefault,
+                        MaterialFlowBasis)
+from idaes.core.util import get_solver
+from idaes.core.util.tables import create_stream_table_dataframe
+from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.util.exceptions import ConfigurationError
+import idaes.core.util.scaling as iscale
+import idaes.logger as idaeslog
+
+__author__ = "Austin Ladshaw"
+
+_log = idaeslog.getLogger(__name__)
+
+# Name of the unit model
+@declare_process_block_class("CoagulationFlocculationZO_JarTestModel")
+class CoagulationFlocculationZO_JarTestModelData(UnitModelBlockData):
```

Comment:
Minor suggestion: I would rename this `CoagulationFlocculationZO` and just describe the model as being attributed to jar tests (which you've done). Later, if you add any subtypes for the model, we could decide whether to make a whole separate unit or declare a config option that would use `jar_test` or other variations.

Reply:
Maybe just take away the 'ZO', since this model is a little more than strictly Zero Order.

Remove the 'JarTest'... can make config options instead...